Problem in make, while building opencv_contrib

I am trying to build opencv_contrib, but it is giving me following error:

/usr/bin/ld: cannot find -lFALSE
/usr/bin/ld: cannot find -lFALSE
collect2: error: ld returned 1 exit status
modules/core/CMakeFiles/opencv_core.dir/build.make:1782: recipe for target 'lib/libopencv_core.so.3.2.0' failed
make[2]: *** [lib/libopencv_core.so.3.2.0] Error 1
make[2]: Leaving directory '/users/anuj.rathore/new/opencv/build'
CMakeFiles/Makefile2:2097: recipe for target 'modules/core/CMakeFiles/opencv_core.dir/all' failed
make[1]: *** [modules/core/CMakeFiles/opencv_core.dir/all] Error 2
make[1]: Leaving directory '/users/anuj.rathore/new/opencv/build'
Makefile:160: recipe for target 'all' failed
make: *** [all] Error 2



How to fix -lFalse ?

--- a/cmake/OpenCVFindLAPACK.cmake
+++ b/cmake/OpenCVFindLAPACK.cmake
@@ -136,6 +136,10 @@ if(WITH_LAPACK)
         unset(LAPACK_LIBRARIES)
         unset(LAPACK_LIBRARIES CACHE)
       endif()
+    else()
+      # LAPACK not found
+      unset(LAPACK_LIBRARIES)
+      unset(LAPACK_LIBRARIES CACHE)
     endif()
   endif()
 

